msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -1454,6 +1452,8 @@ When you use these install and uninstall events, there are some instances where
 
 It's important to know that new events can be added anytime in the future and your bot begins to receive them. So you must design for the possibility of receiving unexpected events. If you're using the Bot Framework SDK, your bot automatically responds with a 200 – OK to any events you don't choose to handle.
 
+## Handling errors in conversation events
```

Comment:
Update it like below:

## Handling errors in conversation events:
When the bot encounters an error while handling different events/activities, We should not post random messages directly to the conversation which has no meaningful context like below:
![image](https://user-images.githubusercontent.com/59052547/178007502-c95b32b6-f895-4930-be29-aea83fdd68d5.png)

So, it's always better to send the meaningful messages in conversations which is specific to the errors for better user experience.
[Handle unexpected errors](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-howto-handle-user-interrupt?view=azure-bot-service-4.0&tabs=csharp#handle-unexpected-errors)

Reply:
@aosolis - Please review my comment and let me know, if you want to add or delete anything from it.